modules/core/tests/client/socket.io.client.service.tests.js

```diff
@@ -1,24 +1,39 @@
-(function() {
+(function(window) {
```

Comment:
Can point me to docs setting out those rules? Why would you pass window when it is not required for the workings of the code?